app/_gateway_entities/route.md

```diff
@@ -4,57 +4,171 @@ content_type: reference
+The following diagram shows how Routes interact with other {{site.base_gateway}} entities:
 
-## Dynamically rewrite request URLs with routes
+{% mermaid %}
+flowchart LR
+  A(API client)
+  B("`Route 
+  (/mock)`")
+  C("`Service
+  (example-service)`")
+  D(Upstream 
+  application)
+  
+  A <--requests
+  responses--> B
+  subgraph id1 ["`
+  **KONG GATEWAY**`"]
+    B <--requests
+    responses--> C
+  end
+  C <--requests
+  responses--> D
 
-Routes can be configured dynamically to rewrite the requested URL to a different URL for the upstream. For example, your legacy upstream endpoint may have a base URI like `/api/old/`. However, you want your publicly accessible API endpoint to now be named `/new/api`. To route the service's upstream endpoint to the new URL, you could set up a service with the path `/api/old/` and a route with the path `/new/api`.
+  style id1 rx:10,ry:10
+  
+{% endmermaid %}
 
-{{site.base_gateway}} can also handle more complex URL rewriting cases by using regular expression capture groups in the route path and the [Request Transformer Advanced](https://docs.konghq.com/hub/kong-inc/request-transformer-advanced/) plugin. For example, this can be used when you must replace `/api/<function>/old` with `/new/api/<function>`.
+For example, if you have an external application and an internal application that need to access the `example_service` Service, but the external application should be limited in how often it can query the Service to assure no denial of service. If a rate limit policy is configured for the Service when the internal application calls it, the internal application is limited as well. Routes can solve this problem.
 
-{{site.base_gateway}} 3.0.x or later ships with a new router. The new router can use regex expression capture groups to describe routes using a domain-specific language called Expressions. Expressions can describe routes or paths as patterns using regular expressions. For more information about how to configure the router using Expressions, see [How to configure routes using expressions](https://docs.konghq.com/gateway/latest/key-concepts/routes/expressions/).
+In the example above, two Routes can be created, say `/external` and `/internal`, and both of them can point to `example_service`. You can configure a policy to limit how often the `/external` Route is used and it can be communicated to the external client for use. When the external client tries to access the Service via {{site.base_gateway}} using `/external`, they're rate limited. But when the internal client accesses the Service using {{site.base_gateway}} using `/internal`, the internal client isn't limited.
```

Comment:
```suggestion

In this example, you can create two Routes to handle the two applications, say `/external` and `/internal`, and point both of them to `example_service`. 
You can configure a policy to limit how often the `/external` Route is used. 
When the external application tries to access the Service via {{site.base_gateway}} using `/external`, they're rate limited. 
But when the internal application accesses the Service using {{site.base_gateway}} using `/internal`, the internal application isn't limited.
```
I removed "and it can be communicated to the external client for use", that seems unnecessary/implied here.